### DISCUSSION TITLE
getShop" reviews properties are not working as expected

### DISCUSSION THREAD

**Discussion by kbasali** [etsy/open-api#775] [Q&A]
Hi Everyone,

Review numbers are not shown correctly with this API call.

Example Etsy Store: AYarnToDyeForDesigns
This Etsy shop has a 1040 Reviews

![image](https://user-images.githubusercontent.com/46492923/209575159-e84b5f42-055d-4292-805f-a5cabcd3c3da.png)

API Results show 344
![image](https://user-images.githubusercontent.com/46492923/209575030-2d29c7c4-b7cd-4a27-ae8a-cc582a1e7f5e.png)

We would be happy to get any suggestions.

**Comment by mjec** [comment]
Hi @kbasali! As mentioned in [the documentation](https://developers.etsy.com/documentation/reference/#operation/getShop), the review count and average returned from `getShop` only include reviews in the past year. The larger number you see on the web is a count of reviews across all time.

**Reply by kbasali** [reply]
@mjec Thank you for your answer. 

Is there any way to get all times reviews?

**Reply by mjec** [reply]
There is no way to fetch aggregate information about all-time reviews at the moment. If that's a feature that would be helpful, let us know a bit about your use case and we'll look into it (of course I can't guarantee it's something we will support in future).

We do have [`getReviewsByListing`](https://developers.etsy.com/documentation/reference/#operation/getReviewsByListing) and [`getReviewsByShop`](https://developers.etsy.com/documentation/reference/#operation/getReviewsByShop) endpoints, which allow you to fetch the content of up to 100 reviews at a time.
